lib/gui/modules/drive-scanner.js

```diff
@@ -16,86 +16,140 @@
 
 'use strict';
 
-const Rx = require('rx');
 const _ = require('lodash');
 const EventEmitter = require('events').EventEmitter;
 const drivelist = require('drivelist');
+const debug = require('debug')('etcher:drive-scanner');
 const settings = require('../models/settings');
 
-const DRIVE_SCANNER_INTERVAL_MS = 2000;
-const DRIVE_SCANNER_FIRST_SCAN_DELAY_MS = 0;
-const emitter = new EventEmitter();
+/**
+ * @summary Time to wait in between scan calls
+ * @type {Number}
+ * @constant
+ */
+const DRIVE_SCANNER_DELAY_MS = 2000;
+
+/**
+ * @summary Time to delay the first scan (in milliseconds)
+ * @description This prevents the drive scans from kicking
+ * off while Etcher is still loading, which would
+ * considerably delay the time until the main window shows
+ * @type {Number}
+ * @constant
+ */
+const DRIVE_SCANNER_FIRST_SCAN_DELAY_MS = 125;
 
-/* eslint-disable lodash/prefer-lodash-method */
+/**
+ * @summary DriveScanner class
+ * @class
+ * @extends {EventEmitter}
+ * @example
+ * var scanner = new DriveScanner()
+ *   .on('error', (error) => { ... })
+ *   .on('drives', (drives) => { ... })
+ */
+class DriveScanner extends EventEmitter {
 
-const availableDrives = Rx.Observable.timer(
-  DRIVE_SCANNER_FIRST_SCAN_DELAY_MS,
-  DRIVE_SCANNER_INTERVAL_MS
-)
+  /**
+   * @summary DriveScanner constructor
+   * @param {Object} [options] - EventEmitter options
+   * @example
+   * var scanner = new DriveScanner()
+   *   .on('error', (error) => { ... })
+   *   .on('drives', (drives) => { ... })
+   */
+  constructor(options) {
+    super(options);
```

Comment:
The only options that would be passed are EventEmitter options, but as it turns out those were removed from the Node docs now, and there's `.setMaxListeners(n)`, which does the same as passing that as option. Removed now.